esmvaltool/recipes/ref/recipe_seaborn_trend_regions.yml

```diff
@@ -0,0 +1,182 @@
+# ESMValTool
+---
+documentation:
+  description:
+    Bo plot, linear trends for IPCC AR6 reference regions.
+    The diagnostic called in this recipe uses the seaborn.boxplot function.
+    See seaborn manual for detailed information
+    (https://seaborn.pydata.org/generated/seaborn.boxplot.html).
+
+  authors:
+    - schlund_manuel
+    - weigel_katja
+  maintainer:
+    - weigel_katja
+  title: Linear trends.
+
+datasets:
+  - {dataset: MIROC6,      project: CMIP6, exp: historical, ensemble: r1i1p1f1, grid: gn}
+
+preprocessors:
+  trend_tas: &trend
+    custom_order: true
+    extract_shape:
+      shapefile: ar6
+      decomposed: true
+      ids:
+        Acronym:
+          - 'NWN'
+          - 'NEN'
+          - 'GIC'
+          - 'WNA'
+          - 'CNA'
+          - 'ENA'
+          - 'NCA'
+          - 'SCA'
+          - 'CAR'
+          - 'NWS'
+          - 'NSA'
+          - 'NES'
+          - 'SAM'
+          - 'SWS'
+          - 'SES'
+          - 'SSA'
+          - 'NEU'
+          - 'WCE'
+          - 'EEU'
+          - 'MED'
+          - 'SAH'
+          - 'WAF'
+          - 'CAF'
+          - 'NEAF'
+          - 'SEAF'
+          - 'WSAF'
+          - 'ESAF'
+          - 'MDG'
+          - 'RAR'
+          - 'WSB'
+          - 'ESB'
+          - 'RFE'
+          - 'WCA'
+          - 'ECA'
+          - 'TIB'
+          - 'EAS'
+          - 'ARP'
+          - 'SAS'
+          - 'SEA'
+          - 'NAU'
+          - 'CAU'
+          - 'EAU'
+          - 'SAU'
+          - 'NZ'
+          - 'WAN'
+          - 'EAN'

```

Comment:
```suggestion
```

If these are all regions you can simply omit `ids`.

Reply:
At the moment these are all land regions (46), if I use:
```
    extract_shape:
      shapefile: ar6
      decomposed: true
      ids:
        Acronym:
```
I get the 15 ocean regions in addition (making the plot even larger and leading to memory issues on Levante even for the 2D variables). (Without (ids: Acronym:) I get the names which are too long to write them on the x-axis ticks.) Not sure, what the best choice is:
- only land (current version)
- land and ocean in one plot or separated
- any other separation or reduction of regions
?

Reply:
Ah, in this case just leave it like this. We let the scientific reviewer decide about the regions 😉 Maybe add a tiny comment that these are all the land regions?

Reply:
Ok, thanks! I added that in the description. And I found the issue with the 3D variables now, I actually forgot to extract the 200hPa layer. Including this, the memory issue is also gone (above that was also caused by a 3D variable, I just did not realize that, sorry)